Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/index/type/enums/ProcessTypeField.java

```diff
@@ -33,6 +33,7 @@
     RULESET("ruleset"),
     BATCHES("batches"),
     COMMENTS("comments"),
+    COMMENTS_MESSAGE("comments.message"),
```

Comment:
I think that in that case you need to make something like comments.id and comments.message. It is probably like JSON standard of ES needs it.